docs/guided-tour/components/helm-chart/README.md

```diff
@@ -0,0 +1,355 @@
+The blueprint is a directory. It must contain a `blueprint.yaml` file. It may contain further files and subdirectories
+like the `deploy-execution.yaml` file in our case.
+
+### Referencing the Helm Chart
+
+Our blueprint defines a Helm deploy item. In the template for the deploy item, the Helm chart must be specified. 
+The preferred way is to reference the corresponding resource in the component version. 
+The entry in the component descriptor looks as follows:
+
+```yaml
+resources:
+  - name: echo-server-chart
+    type: helmChart
+    version: 1.0.0
+    access:
+      type: ociArtifact
+      imageReference: eu.gcr.io/gardener-project/landscaper/examples/charts/guided-tour/echo-server:1.0.0
+```
+
+In the [deploy item template](./blueprint/deploy-execution.yaml), we reference this resource of the component version 
+by its name `echo-server-chart`:
+   
+```yaml
+chart:
+  resourceRef: {{ getResourceKey `cd://resources/echo-server-chart` }}
+```
+
+This snippet shows a Go Templating function `getResourceKey` with a single input argument `cd://resources/echo-server-chart`.
+The result of the Go Templating of this expression is the access data to the helm chart as specified in the component
+descriptor.
+As this function uses ocm to fetch the corresponding resource, you can even switch the storage technology
+(more often referred to as access type in the context of ocm) - thus, e.g. store the helm chart in a helm repository 
+instead of an oci registry - without having to adjust the blueprint (You will of course have to adjust the access in the
+corresponding component version).
+
+>**_NOTE:_**
+> Theoretically, it is possible that the name of a *component reference* or a *resource* is not sufficient to uniquely
+> identify them within a component version. The [ocm specification](https://github.com/open-component-model/ocm-spec/blob/main/doc/01-model/03-elements-sub.md#identifiers)
+> defines that the identity of references as well as resources may optionally also contain a *version* and an
+> *extraId entity*.
+>
+> In such cases, the resource cannot be specified with a path expression. Instead, it has to be specified as defined in
+> the [guidelines of the ocm specification](https://github.com/open-component-model/ocm-spec/blob/main/doc/05-guidelines/03-references.md).
+> The corresponding part of the deploy execution looks like this:
+>
+> ```yaml
+> chart:
+>   resourceRef: {{ getResourceKey `
+>     resource:
+>       name: ocmcli
+>       version: v0.5.0
+>       extraIdentity:
+>         architecture: amd64
+>         os: linux
+> `}}
+> ```
+>
+> Here, you have to be careful with the indentation, as the input string has to be valid yaml. You could also specify
+> this as JSON.
+>
+>```yaml
+> chart:
+>   resourceRef: {{ getResourceKey `{"resource":{"name":"ocmcli","version":"v0.5.0","extraIdentity":{"architecture":"amd64","os":"linux"}}}` }}
+>```
```

Comment:
Should we really add this note to a guided tour?

Reply:
I can remove it in the next pull request.